### PR TITLE
all: Use <CardTitle component="h2"... for cards

### PR DESCRIPTION
**Pull request by KKoukiou**
Cherry-picked out of https://github.com/cockpit-project/cockpit/pull/18739

### CODE REVIEW

File: pkg/storaged/stratis-details.jsx [hunk 1]
```diff
@@ -438,10 +437,8 @@ export const StratisPoolDetails = ({ client, pool }) => {
                     </>
                 ),
             }}>
-                <CardTitle>
-                    <Text component={TextVariants.h2}>
-                        {fmt_to_fragments((pool.Encrypted ? _("Encrypted Stratis pool $0") : _("Stratis pool $0")), <b>{pool.Name}</b>)}
-                    </Text>
+                <CardTitle component="h2">
+                    {fmt_to_fragments((pool.Encrypted ? _("Encrypted Stratis pool $0") : _("Stratis pool $0")), <b>{pool.Name}</b>)}
```
Comment (cockpituous) [inline pkg/storaged/stratis-details.jsx:441]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18746-20230502-181707-d585bac8-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/vdo-details.jsx [hunk 2]
```diff
@@ -266,8 +265,7 @@ export class VDODetails extends React.Component {
                         <StorageButton kind="danger" onClick={delete_}>{_("Delete")}</StorageButton>
                     </>,
                 }}>
-                    <CardTitle><Text component={TextVariants.h2}>{cockpit.format(_("VDO device $0"), vdo.name)}</Text></CardTitle>
-
+                    <CardTitle component="h2">{cockpit.format(_("VDO device $0"), vdo.name)}</CardTitle>
```
Comment (cockpituous) [inline pkg/storaged/vdo-details.jsx:268]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18746-20230502-181707-d585bac8-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/stratis-details.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/storaged/stratis-details.jsx:441]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18746-20230502-194441-84ae16a2-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/vdo-details.jsx (hunk 2 above)
Comment (cockpituous) [inline pkg/storaged/vdo-details.jsx:268]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18746-20230502-194441-84ae16a2-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Comment by martinpitt** [comment]
That's a nice cleanup! Tests need some adjustments, though.

**Review by martinpitt — APPROVED** [review]
If this goes green, please land!

File: pkg/storaged/stratis-details.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/storaged/stratis-details.jsx:441]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18746-20230503-063822-3d050089-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/vdo-details.jsx (hunk 2 above)
Comment (cockpituous) [inline pkg/storaged/vdo-details.jsx:268]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18746-20230503-063822-3d050089-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Pull request merged**